### PR TITLE
Staging dd

### PR DESCRIPTION
**Pull request by speelen**
Minute walk test form names adapted

### CODE REVIEW

**Comment by afolarin** [comment]
are there no changes for VUmC DDs, I see KCL and CIBER are updated?

**Comment by speelen** [comment]
For CIBER this was only a synchronisation with their local version. KCL had a specific request regarding some branching logic. I did not receive such request from VUmc. And since it only affects the branching logic, as far as I am concerned, it is not critical to have it identical for all sites.

File: RADARMSDisVHIRs1/RADARMSDisVHIRs1_DataDictionary.csv [hunk 1]
```diff
@@ -541,7 +541,7 @@ pasat_trial_completed,pasat_msfc,,text,"Para un PASAT 3'' completo, registrar cu
 pasat_trial_no_completed_reasons,pasat_msfc,,radio,"En caso de que no haber finalizado el PASAT 3'' (marcar una opción)","0, Incapaz de finalizar la prueba debido a limitaciones de carácter físico | 1, Otra",,,,,,,,,,,,
 pasat_trial_no_completed_reason_spec1,pasat_msfc,,text,"Especificar las limitaciones físicas",,,,,,,"[pasat_trial_no_completed_reasons] = '0'",,,,,,
 pasat_trial_no_completed_reason_spec2,pasat_msfc,,text,"Especificar otro",,,,,,,"[pasat_trial_no_completed_reasons] = '1'",,,,,,
-six_min_walk_test_score,mwt,,text,"Puntuación en metros",,,,,,,,,,,,,
+six_min_walk_test_score,mw2,,text,"Puntuación en metros",,,,,,,,,,,,,
```
Comment (afolarin) [inline RADARMSDisVHIRs1/RADARMSDisVHIRs1_DataDictionary.csv]: @speelen please confirm that MSDis-VHIR-s1 should not have the 6MWT (looks like the other studies do)

**Comment by speelen** [comment]
This indeed is a mistake. I have updated the Data Dictionary and will create a new PR.

**Pull request merged**